kuma/javascript/src/page.jsx

```diff
@@ -0,0 +1,165 @@
+// @flow
+import * as React from 'react';
+import { useContext } from 'react';
+import { css } from '@emotion/core';
+
+import DocumentProvider from './document-provider.jsx';
+import gettext from './gettext.js';
+import { Row, Strut } from './layout.jsx';
+import Header from './header/header.jsx';
+
+const strings = {
+    relatedTopics: gettext('Related Topics')
+};
+
+const styles = {
```

Comment:
These styles define the high-level structure of an MDN page, but we still rely on main.css and wiki.css to define styles for document content. At this point these styles only support a desktop layout and there is no customization for tablets and phones.